### ISSUE TITLE
Store server.py call times in DB objects

### ISSUE THREAD

**Issue by mwest1066** [PrairieLearn/PrairieLearn#2267]
For debugging and robustness it would be very helpful to track how long function calls are taking. We could store:

* `generate()` time in `variants`
* `render()` time in `view_logs`
* `parse()` time in `submissions`
* `grade()` time in `grading_logs`

etc.

**Comment by eliotwrobson** [comment]
@nwalters512 I think I've seen you report times like these in some newer PRs. Would this be a worthwhile change?

**Comment by mwest1066** [comment]
Agreed on this being low priority, but I think it would be a good thing to add as part of adding better debugging support in general for instructors (@reteps was thinking about this, I believe).
